config/application.rb

```diff
@@ -49,5 +49,7 @@ class Application < Rails::Application
     config.middleware.use 'OliveBranch::Middleware'
 
     config.active_job.queue_adapter = :sidekiq
+
+    config.autoload_paths << "#{Rails.root}/app/models/delegates"
```

Comment:
I think the naming may be breaking the autoloading here. Rails is expecting `models/delegates/user_mvi_delegate` to define `Delegates::UserMviDelegate` and is expecting to find `UserMviDelegate` in `user_mvi_delegate`, not nested. Could move the delegates folder to top level `app/delegates` or rename